src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -332,12 +321,10 @@ private void check() {
                             recurrencePeriod = 0;
                             controller.cleanup(workspace);
                         }
-                        return null;
-                    }
-                }, 10, TimeUnit.SECONDS);
             } catch (Exception x) {
                 LOGGER.log(Level.FINE, "could not check " + workspace, x);
                 ws = null;
+                logger().println("Cannot contact " + node + ": " + x); // TODO as above
```

Comment:
Because the node may simply be offline temporarily. Related to [JENKINS-36013](https://issues.jenkins-ci.org/browse/JENKINS-36013).